plugins/inputs/opcua_client/README.md

```diff
@@ -0,0 +1,57 @@
+# Telegraf Input Plugin: opcua_client
+
+The opcua_client plugin retrieves data from OPCUA slave devices
```

Comment:
Please replace the term `slave` with `replica`, `secondary` or any of the following [suggestions](https://tools.ietf.org/id/draft-knodel-terminology-00.html#suggested-alternatives).

Reply:
Thanks I’ll replace with server to match industry standards nomenclature